docker/compose.testing.yaml

```
@@ -69,21 +69,16 @@
     restart: unless-stopped
-    # https://docs.docker.com/reference/compose-file/services/#volumes
-    volumes:
-      - source: minio_test_data
-        target: /data
-        type: volume
-  postgres_test:
+  postgres-test:
     # https://docs.docker.com/compose/environment-variables/
     # https://docs.docker.com/reference/compose-file/services/#environment
     environment:
         # https://github.com/docker-library/docs/blob/master/postgres/README.md#postgres_db
```

yamllint (1.35.1)

[warning] comment not indented like content

(comments-indentation)